gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -865,6 +873,54 @@ gst_tensor_filter_transform_size (GstBaseTransform * trans,
   return TRUE;
 }
 
+/**
+ * @brief Event handler for sink pad of tensor filter.
+ * @param trans "this" pointer
+ * @param event a passed event object
+ * @return TRUE if there is no error.
+ */
+static gboolean
+gst_tensor_filter_sink_event (GstBaseTransform * trans, GstEvent * event)
+{
+  GstTensorFilter *self;
+  GstTensorFilterPrivate *priv;
+
+  self = GST_TENSOR_FILTER_CAST (trans);
+  priv = &self->priv;
+
+  switch (GST_EVENT_TYPE (event)) {
+    case GST_EVENT_CUSTOM_DOWNSTREAM:
+    {
+      const GstStructure *structure = gst_event_get_structure (event);
+      int ret = -1;
+
+      /** @todo who is supposed to send this event? and how? */
+      if (structure == NULL ||
+          !gst_structure_has_name (structure, EVENT_NAME_UPDATE_MODEL))
+        break;
+
+      if (priv->updatable_model) {
+        const GValue *value = gst_structure_get_value (structure, "model");
```

Comment:
If the filter subplugin does not support update (updateModel is NULL), you will get ret = -1, which makes an error here.

If a filter subplugin does not support, we should not accept set-property, "updatableModel".

Reply:
True, fixed.